cli/rbd/group.py

```diff
@@ -49,6 +49,23 @@ def list(self, **kw):
 
         return self.execute_as_sudo(cmd=cmd)
 
+    def info(self, **kw):
```

Comment:
line 233 also with `info` method?

Reply:
Line 233 info is for Snap class.
The info function i added is for Group Class